charts/trento-server/charts/trento-web/templates/_helpers.tpl

```diff
@@ -85,6 +85,26 @@ Return or generate the grafana admin password
   {{- end -}}
 {{- end -}}
 
+{{- define "trento.web.accessTokenSecret" -}}
```

Comment:
perfect!

Reply:
I'm wondering if there is some other way to reduce this boilerplate, as we have the exact same code for the 3 secrets

Reply:
I have talked with @fabriziosestito , and he already tried to make this prettier, without success, so it is fine as it is.